### PR TITLE
Collections

### PR DESCRIPTION
**Pull request by jayxx1234**
Collections (still needs a bit of work, but pretty good)
Fixes https://github.com/lic-nz/Malone/issues/4

### CODE REVIEW

**Comment by stajs** [comment]
Waiting on #47. Also GitHub says there are conflicts to resolve.

File: LIC.Malone.Client.Desktop/ViewModels/AppViewModel.cs [hunk 1]
```diff
@@ -444,6 +468,8 @@ public AppViewModel()
 			LoadConfig();
 
 			CheckForUpdates();
+
+			Instance = this;
```
Comment (cforlic01) [inline LIC.Malone.Client.Desktop/ViewModels/AppViewModel.cs]: lol we need to fix this

**Comment by stajs** [comment]
#47 resolved, waiting on #68.

**Comment by jayxx1234** [comment]
Woops, didn't mean to merge that in myself (but I resolved the conflicts anyway).

**Comment by stajs** [comment]
It's alright, we'll work through this one next for release. #68 resolved.

**Comment by stajs** [comment]
Canary release up. Noticed one thing:

image

**Comment by jayxx1234** [comment]
The title should now be fixed, and it also doesn't create (or close the Add Collection window) if the collection name is empty or whitespace (as it would not be visible in the list)

**Comment by stajs** [comment]
New canary release for you to test.

**Comment by stajs** [comment]
This will be backed out: https://github.com/lic-nz/Malone/pull/69#issuecomment-183973305

**Pull request merged**